react-native-aztec/ios/RNTAztecView/RCTAztecView.swift

```diff
@@ -29,10 +29,38 @@ class RCTAztecView: Aztec.TextView {
 
     private var previousContentSize: CGSize = .zero
 
+    var textHorizontalInset: CGFloat {
```

Comment:
A suggestion for clarity / readability / maintainability:

```swift
var leftTextInset: CGFloat {
    return contentInset.left + textContainerInset.left + textContainer.lineFragmentPadding
}

var rightTextInset: CGFloat {
    return bounds.width - leftTextInset
}
```

Reply:
Actually, the naming is off... the second one should be `leftTextInsetInRTLMode` or something like that.

Reply:
I mean, naming aside my suggestion would be to have a calculated property for both options.